### PR TITLE
Better detail on the wifi setup

### PR DESCRIPTION
**Pull request by shrung**
Quotation marks are needed for SSIDs which include spaces. The preamble to the code snippet is also grammatically correct now.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @shrung, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;operating-system#1990).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: Documentation/network.md [hunk 1]
```diff
@@ -130,10 +130,10 @@ Home Assistant OS will recreate the default connection profile during boot.
 
 ### Enabling WiFi
 
-WiFi is discouraged for reliability reasons. However if you're still like to use WiFi, you can us the `ha network` command to set up WiFi (example for a Raspberry Pi 4, check `ha network info` to check if your board supports WiFi and the name of the WiFi device):
+WiFi is discouraged for reliability reasons. If you would still like to use WiFi, you can us the `ha network` command to set up WiFi (example for a Raspberry Pi 4, check `ha network info` to check if your board supports WiFi and the name of the WiFi device):
 
 ```bash
-ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid MY-SSID --wifi-psk MY_PASS
+ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid "MY-SSID" --wifi-psk MY_PASS
```
Comment (agners) [inline Documentation/network.md:136]: Ideally we probably also quote the PSK.

```suggestion
ha network update wlan0 --ipv4-method auto --wifi-auth wpa-psk --wifi-mode infrastructure --wifi-ssid "MY-SSID" --wifi-psk "MY-PASS"
```

**Comment by agners** [comment]
Sorry, I just adjusted that documentaitons moments ago :cold_sweat: Can you rebase your change?

File: Documentation/network.md (hunk 1 above)
Comment (shrung) [reply Documentation/network.md:136]: I attempted to use "" on the PSK to test, and the command failed. I think the quotes are not allowed on the PSK (can the PSK contain spaces?)

**Comment by shrung** [comment]
I think I did a fetch/merge? Let me know if more is needed from my end!

**Comment by shrung** [comment]
Okay, I think I did it now, I just resolved it to use your Wi-Fi instead of WiFi

File: Documentation/network.md (hunk 1 above)
Comment (nel) [reply Documentation/network.md:136]: FTR just tested having both with double quote and it's working

Comment (agners) [reply Documentation/network.md:136]: Tested on my end, quotes are working.

**Pull request merged**